SPEC.md

```diff
@@ -10017,7 +10055,9 @@ workflow test_transpose {
   Array[Array[Int]] expected_output_array = [[0, 3], [1, 4], [2, 5]]
   
   output {
-    Boolean is_true = transpose(input_array) == expected_output_array
+    Array[Array[Int]] out = transpose(input_array) 
+    Array[Array[Int]] expected = expected_output_array
```

Comment:
There values will get output but aren't listed in the example output.

Reply:
Fixed